lib/responseDownloadFormats/csv/index.ts

```diff
@@ -0,0 +1,34 @@
+import { createArrayCsvStringifier as createCsvStringifier } from "csv-writer";
+import { ResponseSubmission } from "../types";
+
+export const transform = (responses: ResponseSubmission[]) => {
+  const header = responses[0].submission.map((item) => {
+    return item.questionEn || "";
+  });
+
+  header.unshift("id", "created_at", "confirmation_code");
+
+  const csvStringifier = createCsvStringifier({
+    header: header,
+  });
+
+  const records = responses.map((response) => {
+    return [
+      response.id,
+      response.created_at,
+      response.confirmation_code,
+      ...response.submission.map((item) => {
+        if (item.answer instanceof Array) {
+          return item.answer
+            .map((answer) =>
+              answer.map((subAnswer) => `${subAnswer.questionEn}: ${subAnswer.answer}\n`).join("")
+            )
+            .join("\n");
+        }
+        return item.answer;
+      }),
+    ];
+  });
+
+  return csvStringifier.getHeaderString() + csvStringifier.stringifyRecords(records);
```

Comment:
We ran into a problem in the past with Windows defaulting csv files to ASCII instead of UTF-8. This could be for a future PR if the same issue comes up again:
You can force windows to save a file as UTF-8 with a special character: `"\ufeff"`. E.g.
`return "\ufeff" + csvStringifier.getHeaderString() + csvStringifier.stringifyRecords(records);`

Reply:
Added encoding to the http header which should address, but can revisit later if it comes up in testing